modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsProcessListView.testcase

```diff
@@ -57,12 +57,55 @@ definition {
 	}
 
 	@description = "LPS-88573 - Verify that is possible to order the table items by clicking on the table header Total Pending"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test CanOrderByTotalPendingItems {
+		ApplicationsMenu.gotoPortlet(
+			category = "Workflow",
+			panel = "Applications",
+			portlet = "Process Builder");
+
+		Navigator.gotoNavItem(navItem = "Configuration");
+
+		Workflow.configureWorkflow(
+			workflowDefinition = "Single Approver",
+			workflowResourceValue = "Blogs Entry");
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			panel = "Site Administration",
+			portlet = "Blogs");
+
+		Blogs.addWithWorkflow(
```

Comment:
Can you please replace this with a Blogs API macro? 

JSONBlog.addEntry(
				entryContent = "Blogs Entry Content",
				entryTitle = "Blogs Entry Title");